### DISCUSSION TITLE
Question about Task2 evaluation data

### DISCUSSION THREAD

**Discussion by cuteyyt** [FETS-AI/Challenge#32] [Q&A] [answered]
Dear Organizers:
Thank you very much for your hard work in organizing this challenge.  As validation data (without ground truth) has been released on June 14 for task 1, I have two questions:
1. Is there any validation data for task 2?
2. Can the model for task 2 also submitted to the leaderboard (evaluation applications)?

**Comment by mzenk** [comment, marked as answer]
Hi @cuteyyt,
thank you for your questions!
1. The validation data for task 2 is the same as for task 1. Note, however, that these images come from the same institutions that were included in the training set, while the testing will be done on other institutions.
2. Once the evaluation application is unlocked in IPP, you will be able to submit predictions on the validation set there. It is important to note, though, that the leaderboard created from those for task 2 may not reflect the final challenge ranking, because the validation set essentially measures in-distribution generalization (see point 1).

**Reply by cuteyyt** [reply]
This is really helpful. Thanks!

**Reply by mzenk** [reply]
Great, always happy to answer questions :) 
I'm marking this as answered, feel free to re-open if there are more questions!
